symfony/framework-bundle/5.4/manifest.json

```diff
@@ -9,11 +9,12 @@
     },
     "composer-scripts": {
         "cache:clear": "symfony-cmd",
-        "assets:install %PUBLIC_DIR%": "symfony-cmd"
+        "assets:install %PUBLIC_DIR%": "symfony-cmd",
+        "secrets:generate-keys": "symfony-cmd",
+        "secrets:set APP_SECRET --random": "symfony-cmd"
```

Comment:
I do not know if I should use the `--local` option here

Reply:
doesn't this generate a new APP_SECRET every time?

Reply:
indeed it does :s

2 options:

- either "do not care in dev" that this env is regenerated
- or we should add an option to this command so that, if already existing in the vault, it is not overriden if no flag provided (cf the `secrets:generate-keys --rotate`)

Reply:
@nicolas-grekas friendly ping :)
related to https://github.com/symfony/symfony/issues/38021#issuecomment-685712783 and the creation of the secret logic, do you have an idea here?
many thanks!

Reply:
PR at https://github.com/symfony/symfony/pull/43278 (personal account)